frontend/Dockerfile

```
@@ -21,15 +21,15 @@ RUN usermod --uid $USER_UID node && \
 # so we ensure permissions while we are root
 RUN --mount=type=cache,id=search-a-licious-npm-cache,target=/home/node/.npm,mode=0777 \
   chown -R node:node /home/node/.npm
-COPY --chown=node:node package*.json /opt/search-a-licious
+COPY --chown=node:node package.json package-lock.json /opt/search-a-licious
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)